bxl.sh

```diff
@@ -25,9 +25,15 @@ function findMono() {
 }
 
 function getLkg() {
-    local BUILDXL_LKG_NAME="BuildXL.osx-x64"
-    local BUILDXL_LKG_VERSION=$(grep "BUILDXL_LKG_VERSION" "$MY_DIR/Shared/Scripts/BuildXLLkgVersion.cmd" | cut -d= -f2 | tr -d '\r')
-    local BUILDXL_LKG_FEED_1="https://1essharedassets.pkgs.visualstudio.com/_packaging/BuildXL/nuget/v3/index.json"
+    local LKG_FILE="BuildXLLkgVersionPublic.cmd"
+
+    if [[ -n "$arg_Internal" ]]; then
+        local LKG_FILE="BuildXLLkgVersion.cmd"
+    fi
+
+    local BUILDXL_LKG_VERSION=$(grep "BUILDXL_LKG_VERSION" "$MY_DIR/Shared/Scripts/$LKG_FILE" | cut -d= -f2 | tr -d '\r')
+    local BUILDXL_LKG_NAME=$(grep "BUILDXL_LKG_NAME" "$MY_DIR/Shared/Scripts/$LKG_FILE" | cut -d= -f2 | perl -pe 's/net472/osx-x64/g' | tr -d '\r')
```

Comment:
>perl - [](start = 104, length = 6)

not `egrep`?

Reply:
Tools sometimes slightly differ in behavior on Darwin/BSD in comparison to other systems. I used perl as the expected output should be stable across systems 🐭